pkg/kubectl/cmd/get/init_test.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package get
+
+import (
+	"k8s.io/klog"
+)
+
+func init() {
+	// Enable klog which is used in dependencies
```

Comment:
Which test needs it, why not init in that particular one?

Reply:
- any test needs it so you can trace the events like `go test -race ./pkg/kubectl/cmd/get -v -run='TestWatchOnlyList' -timeout=3s -args -logtostderr=true -v=9`
 - setting a global var from the particular test function seems like a race, given they run in parallel

Reply:
it use to be the default with glog, not with klog